src/api/block.rs

```diff
@@ -278,7 +280,253 @@ fn tokenomics_transaction_operations(block_summary: &BlockSummary) -> Vec<Operat
                     metadata:             None,
                 })
             }
+            SpecialTransactionOutcome::PaydayPoolReward {
+                pool_owner,
+                ..
+            } => {
+                // The events are ordered such that PaydayPoolReward events are followed
+                // by PaydayAccountReward events for the accounts in the given pool.
+                current_pool_owner = *pool_owner;
+            }
```

Comment:
The reason you don't do anything other then record the baker id here is because you keep track of all the "accrue" events already, right?

Reply:
Yes, they're done in the `PaydayAccountReward` events below.